lib/stealth/controller/messages.rb

```diff
@@ -131,16 +135,15 @@ def handle_message(message_tuples)
 
         # Matches the message or the oridinal value entered (via SMS)
         # Ignores case and strips leading and trailing whitespace before matching.
-        def get_match(messages, raise_on_mismatch: true, fuzzy_match: true)
+        def get_match(messages, raise_on_mismatch: true, fuzzy_match: true, slice: true)
```

Comment:
Can we make this a Stealth config setting that defaults to `false` instead? This feels like a breaking change otherwise. The Stealth config would allow us to set this at the bot-level rather than on each call.

It would also limit the amount of method signatures we'd have to change.

Reply:
Great, thanks